wagtail/admin/templatetags/wagtailuserbar.py

```diff
@@ -1,11 +1,15 @@
+from warnings import warn
+
+
 from django import template
 from django.template.loader import render_to_string
 
 from wagtail.admin.userbar import (
     AddPageItem, AdminItem, ApproveModerationEditPageItem, EditPageItem, ExplorePageItem,
     RejectModerationEditPageItem)
-from wagtail.core import hooks
+from wagtail.core import hooks, utils
 from wagtail.core.models import PAGE_TEMPLATE_VAR, Page, PageRevision
+from wagtail.utils.deprecation import RemovedInWagtail210Warning
```

Comment:
Along with the other changes, I think we just need to change this to `RemovedInWagtail211Warning` now, as this will now be in the 2.9 release.